l1-contracts/contracts/state-transition/chain-deps/facets/Admin.sol

```diff
@@ -295,16 +296,21 @@ contract AdminFacet is ZKChainBase, IAdmin {
     /// @inheritdoc IAdmin
     function pauseDepositsAndInitiateMigration() external onlyAdmin onlyL1 {
```

Comment:
while this function is used for migration initiation it actually only pauses, I am not sure the name reflects that

Reply:
https://github.com/matter-labs/era-contracts/pull/1703/commits/ada702bf6c84162bae5dceba47dbae0c851b2915